packages/core/admin/admin/src/content-manager/components/RelationInputDataManager/RelationInputDataManager.js

```diff
@@ -37,10 +37,17 @@ export const RelationInputDataManager = ({
   targetModel,
 }) => {
   const { formatMessage } = useIntl();
-  const { connectRelation, disconnectRelation, loadRelation, modifiedData, slug, initialData } =
-    useCMEditViewDataManager();
+  const {
+    connectRelation,
+    disconnectRelation,
+    loadRelation,
+    slug,
+    initialData,
+    modifiedData,
+    reorderRelation,
```

Comment:
It's not related to your PR, but what do you think about renaming:

- `connectRelation` -> `relationConnect`
- `disconnectRelation` -> `disconnectRelation`
- `reorderRelation` -> `relationReorder`

The benefits would be:

- namespacing and therefore grouped in the code
- it follows the callback prop names `RelationInput` already has

Reply:
yeah i'm not really fussed either way. I'll do it in this PR.